src/main.ts

```diff
@@ -77,10 +80,10 @@ export default class TTSPlugin extends Plugin {
 		this.registerEvent(this.app.workspace.on('editor-menu', ((menu, editor, markdownView) => {
 			menu.addItem((item) => {
 				item
-					.setTitle("Say selected text")
+					.setTitle(window.getSelection().toString().length > 0 ? "Read selected text" : "Read the note")
```

Comment:
Again if user selected a text show "Read selected text" option otherwise show "Read the note" option to read the whole note.